core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts

```diff
@@ -169,6 +180,37 @@ export class TutorCardComponent {
         changes.displayedCard.currentValue)) {
       this.updateDisplayedCard();
     }
+    if (
+      this.platformFeatureService.status.EndChapterCelebration.isEnabled &&
+      this.isOnTerminalCard() &&
+      !this.animationHasPlayedOnce &&
+      this.inStoryMode
+    ) {
+      this.triggerCelebratoryAnimation();
+    }
+  }
+
+  triggerCelebratoryAnimation(): void {
+    this.checkMarkHidden = false;
+    this.checkMarkComponent.animateCheckMark();
+    this.animationHasPlayedOnce = true;
+    let mediaQuery =
+      this.windowRef.nativeWindow.matchMedia('(prefers-reduced-motion)');
```

Comment:
This seems a bit odd, is it possible to use media queries in the CSS to detect this and do the needful? In general, it is good practice to separate template / styling specific code from core logic.

Reply:
Actually, I _could_ use the media query in CSS to prevent the confetti from being animated, but that still doesn't stop the audio clip from being played. The only way to prevent the audio from played is to prevent the function call in the first place, which is why I need to access the media query in the main function.